docs/cli.md

```diff
@@ -135,7 +135,7 @@ The `build-dags` command takes two optional parameters:
 
 * `--start-time` &mdash; the first [runtime](workflow-and-job.md#the-runtime-parameter)
   of your workflows. If empty, a current hour (`datetime.datetime.now().replace(minute=0, second=0, microsecond=0)`)
-  is used for hourly workflows and `datetime.date.today()` for daily workflows.
+  is used.
```

Comment:
why removing this?

Reply:
Because currently, the default start-time value is always date time. There is a user-defined "start_time_factory", which produces a proper start-time.

Reply:
so maybe this should also be updated ?
https://github.com/allegro/bigflow/blob/master/bigflow/cli.py#L325

Reply:
👍 done